minion/consumer_group_offsets.go

```diff
@@ -24,8 +24,14 @@ func (s *Service) ListAllConsumerGroupOffsetsAdminAPI(ctx context.Context) (map[
 	}
 	groupIDs := make([]string, len(groupsRes.AllowedGroups.Groups))
 	for i, group := range groupsRes.AllowedGroups.Groups {
-		if group.GroupState != "Dead" {
+		if len(s.Cfg.ConsumerGroups.ConsumerGroupStates) == 0 {
 			groupIDs[i] = group.Group
+		} else {
+			for _, state := range s.Cfg.ConsumerGroups.ConsumerGroupStates {
```

Comment:
instead of iterating through all states can we keep these states in a map and do a O(1) lookup here?